doc/src/manual/arrays.md

```diff
@@ -764,9 +786,12 @@ julia> speye(3,5)
   [3, 3]  =  1.0
 ```
 
-The [`sparse()`](@ref) function is often a handy way to construct sparse matrices. It takes as
-its input a vector `I` of row indices, a vector `J` of column indices, and a vector `V` of stored
-values. `sparse(I,J,V)` constructs a sparse matrix such that `S[I[k], J[k]] = V[k]`.
+The [`sparse()`](@ref) function is often a handy way to construct sparse array. For
+matrices it takes as its input a vector `I` of row indices, a vector `J` of column
+indices, and a vector `V` of stored values. `sparse(I,J,V)` constructs a sparse matrix
+such that `S[I[k], J[k]] = V[k]`. The equivalent sparse vector constructor is
+[`sparsevec`](@ref), which takes one vector `I` with indices and one vector `V` with the
```

Comment:
No, thats the reason I brought this up in the first place, since it is inconsistent :)
```julia
julia> sparse(rand(3))
3-element SparseVector{Float64,Int64} with 3 stored entries:
  [1]  =  0.521768
  [2]  =  0.794582
  [3]  =  0.690538
```